### ISSUE TITLE
Feature suggestion 

### ISSUE THREAD

**Issue by gguruss** [mixerp/mixerp#99]
After the quotation is sent to client by email, can we implement a feature where the client accepts the quote and is redirected to the application with a payment form filled with the reference number and other information of the quote. And then the client can pay the amount using a pay button.

**Comment by mixerp** [comment]
Yes, it is possible. Do you have any specific process on this? Any standard practices you've come across?

**Comment by gguruss** [comment]
erpNEXT seems to provide this feature.
